app/code/Magento/Checkout/Controller/Cart/Index.php

```diff
@@ -31,6 +32,7 @@ public function __construct(
         \Magento\Store\Model\StoreManagerInterface $storeManager,
         \Magento\Framework\Data\Form\FormKey\Validator $formKeyValidator,
         \Magento\Checkout\Model\Cart $cart,
+        \Magento\Framework\Escaper $escaper,
```

Comment:
this argument must be optional according to parent class.
Please update all other child classes of `\Magento\Checkout\Controller\Cart` with new optional argument